edward/models/dirichlet_process.py

```diff
@@ -51,18 +52,25 @@ def __init__(self, alpha, base_cls, validate_args=False, allow_nan_stats=True,
 
         # Instantiate base distribution.
         self._base = self._base_cls(*self._base_args, **self._base_kwargs)
-        # Define atoms of Dirichlet process, storing only the first as default.
-        self._theta = tf.expand_dims(
-            self._base.sample(self.get_batch_shape()), 0)
+        # Create empty tensor to store future atoms
```

Comment:
use period when ending comments that are sentences, similar to the comment two lines above